R/learner_aorsf_surv_aorsf.R

```diff
@@ -0,0 +1,182 @@
+                                           default = "breslow", tags = "train"),
+                control_cph_eps = p_dbl(default = 1e-9, lower = 0, tags = "train"),
+                control_cph_iter_max = p_int(default = 20L, lower = 1, tags = "train"),
+                control_net_alpha = p_dbl(default = 0.5, tags = "train"),
+                control_net_df_target = p_int(default = NULL, lower = 1L,
+                                              special_vals = list(NULL),
+                                              tags = 'train'),
+                leaf_min_events = p_int(default = 1L, lower = 1L, tags = 'train'),
+                leaf_min_obs = p_int(default = 5L, lower = 1L, tags = 'train'),
+                split_min_events = p_int(default = 5L, lower = 1L, tags = 'train'),
+                split_min_obs = p_int(default = 10, lower = 1L, tags = 'train'),
+                split_min_stat = p_dbl(default = 3.841459, lower = 0, tags = 'train'),
+                oobag_pred_type = p_fct(levels = c("none", "surv", "risk", "chf"),
+                                        default = "surv", tags = 'train'),
+                importance = p_fct(levels = c("none", "anova", "negate", "permute"),
+                                   default = "anova", tags = 'train'),
+                oobag_pred_horizon = p_uty(default = NULL, tags = "train"),
+                oobag_eval_every = p_int(default = NULL, special_vals = list(NULL),
+                                         lower = 1, tags = "train"),
+                attach_data = p_lgl(default = TRUE, tags = "train")
+              )
+
+              super$initialize(
+                id = "surv.aorsf",
+                packages = c("mlr3extralearners", "aorsf", "pracma"),
+                feature_types = c("integer", "numeric", "factor", "ordered"),
+                predict_types = c("crank", "distr"),
+                param_set = ps,
+                properties = c("oob_error", "importance"),
+                man = "mlr3extralearners::mlr_learners_surv.aorsf",
+                label = "Oblique Random Forest"
+              )
+            },
+
+            #' @description
+            #' OOB concordance error extracted from the model slot
+            #' `eval_oobag$stat_values`
+            #' @return `numeric()`.
+            oob_error = function() {
+              nrows <- nrow(self$model$eval_oobag$stat_values)
+              1 - self$model$eval_oobag$stat_values[nrows, 1L]
+            },
+
+            #' @description
+            #' The importance scores are extracted from the model.
+            #' @return Named `numeric()`.
+            importance = function() {
+              if (is.null(self$model)) {
+                stopf("No model stored")
+              }
+              sort(aorsf::orsf_vi(self$model, group_factors = TRUE),
+                   decreasing = TRUE)
+            }
+          ),
+
+          private = list(
+            .train = function(task) {
```

Comment:
Please replace `<-` with `=`, I know this is not typical R, but we prefer to have it consistent.

Reply:
`<-` should be used everywhere now, apart from the places where only `=` works.